lib/modules/apostrophe-events-pages/views/show.html

```diff
@@ -3,7 +3,27 @@
 
 {% block main %}
   <div class="apos-event-preview">
-    <h3 class="apos-event-title">{{ data.piece.title }}</h3>
-    <h5 class="apos-event-date">{{ data.piece.start | date('MMMM Do') }} at {{ data.piece.start | date('h:mma') }}</h5>
+    <h1 class="apos-event-title">{{ data.piece.title }}</h1>
+    <h4 class="apos-event-date">{{ data.piece.start | date('MMMM Do') }} at {{ data.piece.start | date('h:mma') }}</h4>
+    <p>
+    	{{ data.piece.body }}
+    </p>
   </div>
+{% endblock %}
+
+{% block extraHead %}
+<style>
```

Comment:
Again, the style block seems odd. I don't recall seeing this pattern in other modules.

Reply:
backed out work that added filters and a calendar directly into the events-pages module. It just felt like too big a scope. We can make a separate issue to have a directly accessible example in the module if we want.